### DISCUSSION TITLE
Drop all existing tables

### DISCUSSION THREAD

**Discussion by hyochan** [prisma/prisma#2409] [Q&A] [answered]
This feature is useful in testing.
How can I drop all the tables when the test is finished using `@prisma/client`?

**Comment by beeplin** [comment, marked as answer]
if u r using postgresql, you can just `prisma.raw('DROP schema <name> CASCADE')` to drop the whole schema, and then recreat it.

Other wise you can just `DROP DATABASE` and recreate it.

**Reply by revskill10** [reply]
@beeplin Is `prisma` from `import { prisma } from '@prisma/client'` ?

**Reply by jorgeallende** [reply]
I think is from

`import { PrismaClient } from '@prisma/client'`

`const prisma = new PrismaClient();`

**Comment by ludralph** [comment]
Hi @hyochan,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
